README.md

```diff
@@ -18,11 +29,73 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install structured_logger
+    $ gem install logr
 
 ## Usage
 
-TODO: Write usage instructions here
+An example is worth a thousand words they say, so here it is:
+```ruby
+require 'logr'
+
+class YourClass
+  def self.logger
+    @logger ||= Logr::Logger.new('your-logger-name')
+  end
+
+  .
+  .
+  .
+
+  # A complex event that you want to monitor and has metrics associated
+  YourClass.logger.event('event-name', arbitrary: 'context', add_what: 'you_need')
+                  .monitored('Title of the event', 'A longer description.')
+                  .metric('metric-name', 'add as many as you like')
```

Comment:
The metric `metric-name` is not measurable. A metric by definition should be measurable, we cannot record a string as a measure.

I think the concept this line is trying to cover is `tags`. It's actually what the event line has (`arbitrary`, `add_what` are tags with categorical values).

The line should be removed and we should think about tags in general.